objects/rct2tt/ride/rct2.tt.bmvoctps.json

```diff
@@ -141,7 +141,7 @@
             "ru-RU": "Падение из космоса"
         },
         "description": {
-            "en-GB": "A themed rotating observation cabin which travels up a tall tower",
+            "en-GB": "A themed rotating observation cabin",
```

Comment:
It should be something like this "A themed rotating observation cabin shaped like a blob from a sci-fi B-movie"